libcaf_test/caf/test/runnable.hpp

```diff
@@ -62,9 +62,9 @@ class CAF_TEST_EXPORT runnable {
   void info(detail::format_string_with_location fwl, Ts&&... xs) {
```

Comment:
The user-facing function should also be renamed:

```suggestion
  void print_info(detail::format_string_with_location fwl, Ts&&... xs) {
```

Plus adding `print_debug` and `print_warning`.